tsfc/driver.py

```diff
@@ -254,17 +254,25 @@ def facetarea():
     return builder.construct_kernel(kernel_name, body)
 
 
-def compile_expression_at_points(expression, points, coordinates):
+def compile_expression_at_points(expression, points, coordinates, parameters=None):
```

Comment:
I meant that the newly added `parameters` keyword argument is not being used anywhere yet. This function is called [here](https://github.com/firedrakeproject/firedrake/blob/master/firedrake/interpolation.py#L168) for [interpolating UFL expressions](http://www.firedrakeproject.org/interpolation.html#ufl-expressions).